src/DotNet.Testcontainers/Containers/Modules/Databases/CouchbaseTestcontainer.cs

```diff
@@ -14,19 +16,14 @@ internal CouchbaseTestcontainer(ITestcontainersConfiguration configuration) : ba
 
     public async Task<long> CreateNewBucket(string bucket, int memory = 128)
     {
-      var createBucketCommand =
-        "bash /opt/couchbase/bin/couchbase-cli bucket-create -c 127.0.0.1:8091 " +
-        $"--username {this.Username} --password {this.Password} " +
-        $"--bucket={bucket} --bucket-type couchbase --bucket-ramsize {memory.ToString()} --enable-flush 1 --bucket-replica 0";
-      return await this.ExecAsync(new[] {"/bin/sh", "-c", createBucketCommand});
+      var createBucketCommand = $"{couchbaseCli} bucket-create -c 127.0.0.1:8091 --username {this.Username} --password {this.Password} --bucket={bucket} --bucket-type couchbase --bucket-ramsize {memory} --enable-flush 1 --bucket-replica 0";
```

Comment:
@HofmeisterAn what do you think to pass --bucket-type with a parameter which has default value "couchbase" like following;
CreateNewBucket(string bucket, int memory = 128,string bucketType=couchbase)

Reply:
I was also thinking about that. We can changes this with a new feature.